src/meracache.jl

```diff
@@ -136,7 +145,8 @@ See also: [`set_stored_densitymatrix!`](@ref), [`has_densitymatrix_stored`](@ref
 function get_stored_densitymatrix(c::MERACache{N}, depth) where N
     depth = Int(min(N, depth))
     rho = c.densitymatrices[depth]
-    return rho::operatortype(c)
+    @assert rho !== nothing
+    return rho#::operatortype(c)
```

Comment:
Was the comment added before you were sure that this is type stable without it? Can it be removed now?